src/header/utils.js

```diff
@@ -64,9 +64,16 @@ export const getToolsMenuItems = ({ studioBaseUrl, courseId, intl }) => ([
     title: intl.formatMessage(messages['header.links.import']),
   },
   {
-    href: `${studioBaseUrl}/export/${courseId}`,
-    title: intl.formatMessage(messages['header.links.export']),
-  }, {
+    href: `${studioBaseUrl}/import/${courseId}`,
```

Comment:
This path seems to point to `import`, however the title seems to be for export.

Reply:
Thank you @xitij2000 !
Fixed here: d0b1bec109f6c6e266653c3a2f42d74611b05124